src/packaging/docker-services/generate-ssl-stores/docker-entrypoint.sh

```diff
@@ -49,14 +49,21 @@ GENERIC_TRUSTSTORE=${WORKDIR}/ssl-stores/generic-server-truststore.jks
 
 CA_CERT_CONFIG=${WORKDIR}/ca_cert.conf
 ROOT_CA_CERT=${WORKDIR}/ssl-stores/ca-cert
+CA_KEY=${WORKDIR}/ca-key
+
+if [ -f "$CA_CERT_CONFIG" ] ; then
```

Comment:
Remove the error checking here.

Similar to my previous comment about the container being an example. The idea is to put the minimal amount of code to show the steps on how to produce the artefacts.